doc/adr/0006-tsp-offset-traces-in-experiment.md

```diff
@@ -0,0 +1,42 @@
+# 4. Tsp update traces in experiment
+
+Date: 2022-02-22
+
+## Status
+
+Proposed. WIP
+
+## Context
+
+The TSP specification [TSP specification][tsp] proposes a PUT command on the tsp/experiments/{expUUID} endpoint for the following use cases:
+
+- Rename experiment
+- Add/remove of traces to/from an experiment
+- Other updates (e.g. offsetting of traces)
+
+This ADR proposes the way-forward for the adding/removing of traces in an experiments.
```

Comment:
in an experiment.

Also, I think this sentence doesn't belong to this specific ADR.